Makefile

```diff
@@ -0,0 +1,49 @@
+KUEUE_REPO=https://github.com/kubernetes-sigs/kueue.git
+KUEUE_TMP_PATH=/tmp/xpk_tmp/kueue
```

Comment:
```suggestion
KUEUE_TMP_PATH=$(PROJECT_DIR)/tmp
```

Reply:
I believe putting sth that will be cloned into and than removed into /tmp makes more sense than putting it in xpk/tmp and creating this directory

Reply:
Make sense.